assingments/mergesort.py

```diff
@@ -0,0 +1,55 @@
+#import numpy
```

Comment:
please change directory to /assignments/linsi/mergesort.py

Reply:
changed it